templates/watchdog.conf.j2

```diff
@@ -10,7 +10,7 @@ admin              = {{watchdog_admin}}
 interval           = {{watchdog_interval}}
 {% endif %}
 {% if watchdog_realtime %}
-realtime           = {{watchdog_realtime}}
+realtime           = {{ "yes" if watchdog_realtime == True else watchdog_realtime }}
```

Comment:
How about `"yes" if watchdog_realtime else "no"`?

Reply:
Then `watchdog_realtime: "no"` (a string that will evaluate to True) will result in `realtime = "yes"`. If Only We Had Static Types...

Reply:
Ah, good point. Thanks for clarifying.